_test/general.test.php

```diff
@@ -5,13 +5,68 @@
  * @group plugin_findologicxmlexport
  * @group plugins
  */
-class general_plugin_findologicxmlexport_test extends DokuWikiTest {
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+
+class general_plugin_findologicxmlexport_test extends DokuWikiTest
+{
+    /**
+     * A page needs to have content. This is the placeholder for this value.
+     */
+    const PAGE_CONTENT_PLACEHOLDER = 'This page is for test purposes.';
+
+    /**
+     * Helper function to save a page.
+     *
+     * @param array $ids An array with page ids (eg. start or wiki:syntax)
+     * @param string $content Optional content, else the default test value will be set
+     */
+    private function savePages($ids, $content = self::PAGE_CONTENT_PLACEHOLDER)
+    {
+        foreach ($ids as $id) {
+            saveWikiText($id, $content, '');
+            idx_addPage($id);
+        }
+    }
+
+    /**
+     * Gets parameters, calls the Export and returns the XML as SimpleXMLElement.
+     *
+     * @param int $start Optional start value, default 0 will be set
+     * @param int $count Optional count value, default 20 will be set
+     * @param array $conf Optional configuration, default empty array will be set
+     * @return SimpleXMLElement Export generated XML
+     */
+    private function getXML($start = 0, $count = 20, $conf = array())
+    {
+        $DokuwikiXMLExport = new DokuwikiXMLExport($conf);
```

Comment:
Per convention variables start with a lower-case letter.

Reply:
I hate it when such errors happen to me :-)